lib/shared/bucketing-assembly-script/assembly/bucketing/segmentation.ts

```diff
@@ -18,18 +19,20 @@ import { getF64FromJSONValue } from '../helpers/jsonHelpers'
  * Returns true if the user's data allows them through the segmentation
  * @param operator - The set of filters to evaluate, and the boolean operator to follow (AND, OR, XOR)
  * @param user - The incoming user, device, and user agent data
+ * @param audiences - a map of audience_id to audience, used for audienceMatch filters
  */
 export function _evaluateOperator(
     operator: TopLevelOperator,
-    user: DVCPopulatedUser
+    user: DVCPopulatedUser,
+    audiences: Map<string, Audience>
```

Comment:
if we follow Nika's config stuff, this would technically not have an _id field in the Audience